kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html

```diff
@@ -333,14 +347,21 @@
                             </span>
 
                             <span class="editable-empty" ng-if="!ctrl.metadata.credit && ctrl.userCanEdit">Unknown (click ✎ to add)</span>
+                            <span class="editable-empty" ng-if="!ctrl.metadata.credit && !ctrl.userCanEdit">Unknown</span>
                         </span>
                     </span>
                 </span>
 
             </dd>
 
-            <dt ng-hide="locationEditForm.$visible" class="image-info__wrap metadata-line metadata-line__key image-info__group--dl__key--panel">Location</dt>
-            <dd class="image-info__wrap metadata-line metadata-line__info image-info__group--dl__value--panel">
+            <dt ng-if="(ctrl.hasLocationInformation() || ctrl.hasMultipleValues(ctrl.rawMetadata.subLocation) ||
+                            ctrl.hasMultipleValues(ctrl.rawMetadata.city) ||
+                            ctrl.hasMultipleValues(ctrl.rawMetadata.state) ||
+                            ctrl.hasMultipleValues(ctrl.rawMetadata.country)) || ctrl.userCanEdit" ng-hide="locationEditForm.$visible" class="image-info__wrap metadata-line metadata-line__key image-info__group--dl__key--panel">Location</dt>
+            <dd ng-if="(ctrl.hasLocationInformation() || ctrl.hasMultipleValues(ctrl.rawMetadata.subLocation) ||
+                            ctrl.hasMultipleValues(ctrl.rawMetadata.city) ||
+                            ctrl.hasMultipleValues(ctrl.rawMetadata.state) ||
+                            ctrl.hasMultipleValues(ctrl.rawMetadata.country))  || ctrl.userCanEdit" class="image-info__wrap metadata-line metadata-line__info image-info__group--dl__value--panel">
```

Comment:
I'm probably missing something - what is the reason for checking `ctrl.hasMultipleValues` on the `city`, `state`, and `country` separately to `hasLocationInformation()`? Is the location information returned on `rawMetadata` different from `metadata`?

Reply:
Sorry, but the honest answer is: I don’t know. This was kindly suggested by @andrew-nowak after it wasn’t working correctly before. All fault mine, obviously…

Reply:
No worries, I thought I'd check since I understood `ctrl.hasMultipleValues` was being used elsewhere to display `Multiple something` in the field but didn't seem like that's what we want for the location fields

Reply:
thanks! the difference (_I think_) is that previously the location field was always shown, whereas we now want to not show the location field if there's no data and the user cannot edit. In an ideal world the `hasLocationInformation` would reliably tell us if there's location info or not, but currently it only works on single image selection. We could update it to understand multiple image selection, but it is also used elsewhere and we don't want/have time to untangle right now.

(The highlighted code is similar to the existing https://github.com/guardian/grid/blob/ca2628a362da5b9de8ffcb0dcd38f3c52f9b3820/kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html#L372-L377)